### PR TITLE
MEPTS-1356 | TXML CHANGES DECEMBER

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java [hunk 1]
```diff
@@ -857,96 +861,70 @@ public static String getPatientsTracedWithVisitCard(
...
-            + "                       o1.concept_id = ${returnVisitDateForArvDrugConcept} and o1.voided=0 "
-            + "               where  o1.obs_id is null "
-            + "           ) fila ON ficha.patient_id=fila.patient_id ) filaficha on filaficha.patient_id=pm.patient_id "
-            + "       )ps "
-            + "       GROUP  BY ps.patient_id";
+        "SELECT pat.patient_id "
+            + "FROM   patient pat "
+            + "WHERE  pat.voided=0 "
+            + "    AND    pat.patient_id NOT IN( "
+            + "        SELECT fn.patient_id "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java]: This already exists in txcurr

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
See the comments

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java]: just used the same implementation! Please check

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**